Code/Framework/AzToolsFramework/AzToolsFramework/ViewportUi/ViewportUiSwitcher.h

```diff
@@ -40,6 +40,9 @@ namespace AzToolsFramework::ViewportUi::Internal
         void SetButtonTooltip(ButtonId buttonId, const AZStd::string& tooltip);
 
     private:
+        //! Removes color from switcher icon.
```

Comment:
nit: Also I guess this could remove color from any icon? Not just the switcher 🙂